pkgs/development/libraries/libgpod/default.nix

```diff
@@ -1,18 +1,20 @@
-{stdenv, lib, fetchurl, gettext, perlPackages, intltool, pkgconfig, glib,
+{ stdenv, lib, fetchurl, gettext, perlPackages, intltool, pkgconfig, glib,
   libxml2, sqlite, zlib, sg3_utils, gdk-pixbuf, taglib,
-  libimobiledevice, python3Packages, mutagen,
+  libimobiledevice, mutagen,
   monoSupport ? false, mono, gtk-sharp-2_0
 }:
 
-let
-  inherit (python3Packages) python pygobject2;
```

Comment:
Though this is actually pygobject2 on Python 3.

Reply:
Oh, I didn't even notice this. And pygobject2 did support python3.